libeos-payg/tests/boottime-source.c

```diff
@@ -40,11 +40,16 @@ timeout_cb (gpointer data)
 static void
 test_once (void)
 {
-  g_autoptr(GSource) source = epg_boottime_source_new (INTERVAL_MS);
+  g_autoptr(GError) local_error = NULL;
+  g_autoptr(GSource) source = NULL;
   guint called = 0;
   gint64 start = epg_get_boottime ();
   gint64 end;
 
+  source = epg_boottime_source_new (INTERVAL_MS, &local_error);
+  g_assert (source != NULL);
```

Comment:
`g_assert_nonnull()`, and do the `g_assert_no_error()` call first to get more detailed error messages out on failure.